content/md/assets/full-automatic-pam-migration.md

```diff
@@ -0,0 +1,112 @@
+---
+id: full-automatic-pam-migration
+title: The **full-automatic** migration guide for PAM assets
+popular: false
+ee-only: true
+related: from-PAM-to-new-AM, how-to-migrate-from-pam
+---
+
+# What's the full-automatic migration?
+In this migration, **everything is automated**, you just need to launch one single command and your PAM assets will be migrated in the 4.0 new Asset Manager:
```

Comment:
I would add also a warning about the fact that they should not use their PIM before migrating their assets as they may lose data.

Reply:
We will see with samir if there is not another way to avoid that and make sure that we don't loose any data